### ISSUE TITLE
Game Pause Crash (Solution found)

### ISSUE THREAD

**Issue by rdunlap** [TanninOne/modorganizer#441]
Ok, I have tested as well as I can so I am going to throw it out for discussion. I have an issue that is "Apparently" related to MO2. Yesterday I paused my game to go and answer the door and it took me a few minutes to get back to the game. When I returned Fallout was crashed. NOTE: MO2 was not crashed. This is consistent behavior now. I can't say if it was happening before because I never leave it paused more than a minute or two. I cant say if Alpha 1 showed this behavior. I am pointing to MO2 because after verifying that it was happening consistently I went to the Fallout launcher and loaded the vanilla game. No crash. Then I went into MO2 and unchecked all my mods closed and re-opened and started the game. Even with no mods if MO2 is involved 8-10 minutes paused and it crashes. I don't see anything in the logs other than a generic app crash and no indication in MO2.

Anyone have any ideas?

Robert

**Comment by GSDFan** [comment]
I am assuming MO2 alpha2 and fallout 4. I was paused for 25 minutes and no crash.

**Comment by rdunlap** [comment]
yes, MO2 A2 and its strange I don't truly suspect MO2 at this point but am trying to be diligent. MO2 has been too stable so far and not shown anything like this. Problem is that I haven't changed anything other than a couple of mods and they don't seem to make any difference. Thanks for trying to reproduce though. It helps direct me to something else. Maybe something in the patch Bethesda pushed out is causing an interaction issue of some kind. I am going to leave it open a couple of days and see if anyone else has any luck reproducing it, if not then it has to be something on my particular setup causing it. 

NOTE: I did do a clean installation of MO2A2 and its behaving the same way. (I didn't go into the registry and do any cleanup since I couldn't think of anything there that could cause this) In the MO2 log files the last entry has been a windows error indication but this has been there since Alpha 1 and doesn't seem to be a real error since windows don't record one to correspond to it. Seems like MO2's way to say windows reports that Fallout stopped.

**Comment by rdunlap** [comment]
GSDFan Thanks for trying it on your rig. Turns out that the ENB I was using was not happy when Windows slept the monitors. If MO2 was not involved it handled it but with MO2 running it runs home to mommy. I updated from the Beta to the .288 binary and its working like I expected. I guess i hadn't let it stay paused long enough before to trigger the event. At this point I am not going to call it a bug in MO2 or in ENB. It was a simple interaction issue that went away when he moved out of beta with ENB.

Robert
